tests/TestCase/I18n/DateTest.php

```diff
@@ -163,6 +163,23 @@ public function testJsonSerialize($class)
         $this->assertEquals('"2015-11-06T00:00:00+00:00"', json_encode($date));
     }
 
+    /**
+     * Tests change json encoding format
+     *
+     * @dataProvider classNameProvider
+     * @return void
+     */
+    public function testSetJsonEncodeFormat($class)
+    {
+        $date = new $class('2015-11-06 11:32:45');
+
+        $class::setJsonEncodeFormat(DATE_ATOM, false);
+        $this->assertEquals('"2015-11-06T00:00:00+00:00"', json_encode($date));
+
+        $class::setJsonEncodeFormat("yyyy-MM-dd'T'HH':'mm':'ssZZZZZ");
```

Comment:
Is it unwise to change the default formatting to use the timezone specifier that works with a wider set of icu versions?

Reply:
@markstory Yes it would be unwise. Changing the default format could break API consumers which expect the datetime string in specific format.